src/lib/time.ts

```diff
@@ -1,17 +1,25 @@
-import type { DateStringOptions } from "./types";
-
 export const lastWeek = new Date(new Date().getTime() - 7 * 24 * 60 * 60 * 1000);
 export const lastMonth = new Date(new Date().getTime() - 31 * 24 * 60 * 60 * 1000);
 export const lastSixMonths = new Date(new Date().getTime() - 31 * 24 * 60 * 60 * 1000 * 6);
 export const lastYear = new Date(new Date().getTime() - 365 * 24 * 60 * 60 * 1000);
 export const all = new Date('2022-12-31');
 
-export function formatISODate(date: string, { weekday, year, month, day }: DateStringOptions) {
+export function formatISODate(
+	date: string,
+	{ weekday, year, month, day }: Intl.DateTimeFormatOptions
+) {
 	return new Date(date).toLocaleDateString('en-us', {
-		timeZone: "UTC",
+		timeZone: 'UTC',
 		weekday,
 		year,
 		month,
 		day
 	});
 }
+
+export function secondsBeforeCronTrigger() {
+	const date = new Date();
+	if (date.getUTCHours() >= 16) date.setUTCDate(date.getUTCDate() + 1)
```

Comment:
Is this 16 coming from +/-4 offset?

Reply:
Yeah, using UTC time to have things be consistent since getHours() and setHours() depend on local time of wherever the code is being run. In UTC time the cron trigger is 4 pm (16 hrs through the day I think)